### ISSUE TITLE
beach init is broken

### ISSUE THREAD

**Issue by kdambekalns** [flownative/localbeach#18]
```
$ beach version
Local Beach v1.3.1+1 © 2019-2020 Robert Lemke / Flownative GmbH

$ beach init
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x136b844]

goroutine 1 [running]:
github.com/flownative/localbeach/cmd/beach/cmd.handleInitRun(0x1765720, 0x1788638, 0x0, 0x0)
	/home/runner/work/localbeach/localbeach/cmd/beach/cmd/init.go:51 +0x644
github.com/spf13/cobra.(*Command).execute(0x1765720, 0x1788638, 0x0, 0x0, 0x1765720, 0x1788638)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:830 +0x2aa
github.com/spf13/cobra.(*Command).ExecuteC(0x1765ea0, 0x103af5a, 0x172a820, 0xc000044750)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914 +0x2fb
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
github.com/flownative/localbeach/cmd/beach/cmd.Execute()
	/home/runner/work/localbeach/localbeach/cmd/beach/cmd/root.go:33 +0x31
main.main()
	/home/runner/work/localbeach/localbeach/main.go:8 +0x20
```

**Comment by kdambekalns** [comment]
Added error handling and changes in https://github.com/flownative/localbeach/commit/7a885ef31fbed889dbcc8d8af2d3bfbe6e1af5d7 cuased this. Result: `sandbox` in https://github.com/flownative/localbeach/blob/433def840178ca80211e9b792780f442ca9b3db5/cmd/beach/cmd/init.go#L51 is `nil`

**Comment by kdambekalns** [comment]
Fixed with v1.3.2+1
